langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/internal/chat/JsonSchemaElement.java

```diff
@@ -10,13 +11,19 @@
 public abstract class JsonSchemaElement {
 
     @JsonProperty
-    private final String type;
+    @JsonFormat(with = {JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY,
+            JsonFormat.Feature.WRITE_SINGLE_ELEM_ARRAYS_UNWRAPPED})
+    private final String[] type;
```

Comment:
This is an odd change, but I don't know how else to do it. Anyway, with `WRITE_SINGLE_ELEM_ARRAYS_UNWRAPPED`, it does not affect the (more common) case where there is just one type.